cyhy/mailer/CyhyMessage.py

```diff
@@ -29,8 +29,19 @@ class CyhyMessage(ReportMessage):
 
     TextBody = """Greetings {{acronym}},
 
-The Cyber Hygiene scan results are attached for your review. Same password as before. (If this is your first report and you have yet to receive a password, please let us know!)
+The Cyber Hygiene scan results are attached for your review. Same password as before. (If this is your first report and you have yet to receive a password, please let us know.)
 
+{{#has_tech_pocs}}
+Note: CISA Assessments has the following information listed as the Technical Points of Contact for {{acronym}}:
```

Comment:
Are we migrating to CISA Cyber Assessments now? (same question for all uses of CISA Assessments)

Reply:
Waiting for clarification from the CYHY team in NCATS JIRA ticket CYHYDEV-733.

Reply:
I changed "CISA Assessments" to "NCATS" for now.  We can update the email body once we have updated the language in the reports.

See 3ebaf82.